pywhat/regex_identifier.py

```diff
@@ -16,8 +16,15 @@ def check(self, text, distribution: Optional[Distribution] = None):
             distribution = self.distribution
         matches = []
 
+        with open("pywhat/Data/tld_list.txt", "r") as file:
+            tlds = file.read()
```

Comment:
Can we make the python script do this, so we don't do it everytime we load it? I'd rather it all be in one regex in the fiile to start with :-)

Reply:
Ooo that's a good idea!!

Reply:
I have changed this now. The script is working but the regex is hardcoded there, so if we change it in the database it needs to be changed in the script too. If I can do this somehow differently, let me know :sweat_smile:

Reply:
nah nah hard-coding URL regex is how i would do it too :-)